tests/micropython/run_all_tests.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python3
+#LVGL MicroPython binding batch-test  (iterates through all files in the test-case directory)
+
+import sys
+import os
+import glob
+
+sys.dont_write_bytecode = True
+import test_constants as lv_const
+
+
+class Var:
+    MICROPYTHON_COMMAND = "./"  #"../../../../../../lv_micropython/ports/unix/build-standard/micropython "
```

Comment:
The path to MicroPython (unix) executable should come from environment variable or as an argument of this script. But not hard-coded path.
Please check [argparse Python module](https://docs.python.org/3/library/argparse.html) for easy usage of arguments.

Example:
```
import argparse

parser = argparse.ArgumentParser(prog="run_all_tests", description="Run all MicroPython tests")
parser.add_argument("-m", "--micropython", help="Path to MicroPython unix executable")
args = parser.parse_args()

# Access argument value:
print("Path: ", args.micropython)
```

Reply:
Using default value:
```
parser.add_argument("-m", 
                                   "--micropython", 
                                   default="../../../../../ports/unix/build-standard/micropython",
                                   help="Path to MicroPython unix executable")
```

Reply:
If we run the test from `lv_micropython`, I think the same argument applies for [here](https://github.com/lvgl/lvgl/pull/4645#discussion_r1355806689).